From giovanni at paclan.it  Fri Jul  1 09:35:49 2016
From: giovanni at paclan.it (Giovanni)
Date: Fri, 1 Jul 2016 07:35:49 +0000 (UTC)
Subject: dkim not working after upgrading to 2.11.0
Message-ID: <nl56gl$f40$1@ger.gmane.org>

Hi,
after upgrading to 2.11.0, DKIM signing does not work anymore because
the email is no more detected as ORIGINATING.
This diff fixes the issue.
 Cheers
  Giovanni


--- amavisd.orig    Tue Apr 26 21:24:33 2016
+++ amavisd Fri Jul  1 01:03:15 2016
@@ -34338,6 +34329,7 @@ sub collect_some_dkim_info($) {
     $sig_ind++;
   }
   Amavis::load_policy_bank($_,$msginfo) for @bank_names;
+  $msginfo->originating(c('originating'));
   $msginfo->dkim_signatures_valid(\@signatures_valid)  if @signatures_valid;
 # if (ll(5) && $sig_ind > 0) {
 #   # show which header fields are covered by which signature
